React/Modules/RCTClipboard.m

```diff
@@ -22,6 +22,24 @@ - (dispatch_queue_t)methodQueue
   return dispatch_get_main_queue();
 }
 
+- (NSArray<NSString *> *)supportedEvents
+{
+  return @[@"clipboardChanged"];
+}
+
+- (void)startObserving {
+  NSNotificationCenter *nc = [NSNotificationCenter defaultCenter];
+  [nc addObserver:self selector:@selector(clipboardChanged:) name:UIPasteboardChangedNotification object:[UIPasteboard generalPasteboard]];
+  [nc addObserver:self selector:@selector(clipboardChanged:) name:UIPasteboardRemovedNotification object:[UIPasteboard generalPasteboard]];
+}
+
+- (void)stopObserving {
```

Comment:
Both `startObserving` and `stopObserving` will be called from `RCTEventEmitter`

From the `RCTEventEmitter.h`:
```
/**
 * These methods will be called when the first observer is added and when the
 * last observer is removed (or when dealloc is called), respectively. These
 * should be overridden in your subclass in order to start/stop sending events.
 */
- (void)startObserving;
- (void)stopObserving;
```